src/MainWindow.cpp

```diff
@@ -1531,3 +1549,48 @@ QMessageBox::StandardButton MainWindow::openNewFileQuestionDialog(const QString&
           QMessageBox::Yes | QMessageBox::No, QMessageBox::No
     );
 }
+
+// TODO: Reuse code between generateBashRemediationRole and generateAnsibleRemediationRole
+void MainWindow::generateBashRemediationRole()
+{
+    const QString filename = QFileDialog::getSaveFileName(this,
+        QObject::tr("Save as bash script"),
+        QObject::tr("remediate-profile.sh"),
```

Comment:
Could we please base the suggested file name on the benchmark and profile ID? that would increase UX because many times I can just click save without typing anything.

Reply:
I keep this in mind, but I have postponed it. Do you have any idea what is a good form? I am not sure about benchmark IDs, but those profile IDs alone are quite long.